tasks/translations_terms.md

```diff
@@ -0,0 +1,21 @@
+* **@mention**: see **mention**.
+* **/me**: see **me**.
+* **customization**: changing Zulip's settings so that it behaves in ways you prefer.
+* **emoji**: small image used in chat messages, for terse non-verbal expression or cuteness; loanword into English from Japanese 絵文字.
+* **filter**: one of the options for viewing different kinds of messages, listed in the upper left corner of the window: Home, Private messages, Starred messages, @-mentions. Also called a "view".
+* **group PM**: a private message sent directly to two or more other users rather than through a stream.
+* **home**: a "view" in which all topics in all subscribed streams are visible, in order by date and time of posting
+* **me**: a reference to one's own name, formatted as `/me`.
```

Comment:
I guess in theory we could support this being translated, though for now maybe we should mention that it shouldn't be?